mlkem/indcpa.c

```diff
@@ -405,9 +405,10 @@ __contract__(
   }
 }
 
-
-
-STATIC_ASSERT(NTT_BOUND + MLKEM_Q < INT16_MAX, indcpa_enc_bound_0)
+#if NTT_BOUND > INT16_MAX - MLKEM_Q
+#error \
+    "The bound for the forward NTT is too large to ensure non-overflow in indcpa_keypair_derand"
+#endif
```

Comment:
Isn't this something that the CBMC proofs should garuantee now? 
I'm tempted to remove all such bound assertations.

Reply:
Yes, CBMC checks the same. Ok, let's remove it.

Reply:
Done

Reply:
thx